templates/error/descriptions/E0001.md.hbs

```diff
@@ -17,9 +17,26 @@ until its configuration is changed.
 The problem is typically one of these:
 
 1. The server was started with **the wrong hostname.**
-    - Use the `--baseUrl` (or `-b`) option to set the base URL.  
+    - Use the `--baseUrl` (or `-b`) option to set the base URL.
       Example: `community-solid-server --baseUrl https://mydomain.example/`
 
 2. A **reverse proxy** running in front of the server passes the wrong headers.
     - Set up the [`Forwarded` header](https://solidproject.org/self-hosting/css/nginx)
       to pass the correct URL on to the server.
+
+## Circumventing this for development
+
+For development reasons it might be easier to temporarily allow requests from multiple host locations.
+This can be done by adding the following lines to your server configuration,
+forcing the server to always assume the host header is the one defined by the `--baseUrl` parameter.
```

Comment:
```suggestion
For development reasons, it can be easier to temporarily allow requests from multiple host locations.
This can be done by adding the following lines to your server configuration,
causing the server to always assume the value of the `host` header is defined 
by the `--baseUrl` parameter.
```